backend/kernelCI_app/views/treeDetailsSlowView.py

```diff
@@ -242,7 +242,10 @@ def __processBootsTest(self, currentRowData):
             self.bootFailReasons[testError] = self.bootFailReasons.get(testError, 0) + 1
 
         if testEnvironmentCompatible is not None:
-            self.bootEnvironmentCompatible[testEnvironmentCompatible] += 1
+            if testStatus is None:
+                self.bootEnvironmentCompatible[testEnvironmentCompatible]["NONE"] += 1
```

Comment:
very nit: but  you can call it "NULL"

![image](https://github.com/user-attachments/assets/cb5ae692-79d8-4401-bf8e-747e1ae9594d)